targeted/json_csrf_poc_generator.js

```diff
@@ -4,6 +4,11 @@
 // released under the Apache v2.0 license.
 //You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 //Author : @haseebeqx
+const CONTENT_TYPE = Java.type(
```

Comment:
I'd have expected the Java.type on the HttpHeader (more common usage and avoids loading the class more than once like in the xml script).